src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -506,7 +506,7 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_arcsinh)
 
 // arccosh
 MXNET_OPERATOR_REGISTER_UNARY(arccosh)
-.describe(R"code(Inverse hyperbolic cosine, element-wise.
+.describe(R"code(Returns element-wise inverse hyperbolic cosine value of the input.
```

Comment:
"Returns the element-wise hyperbolic cosine of the input array, computed element-wise."